test/vcf/debugulator_integration_test.cpp

```diff
@@ -40,7 +38,7 @@ namespace ebi
       std::vector<std::unique_ptr<ebi::vcf::ReportWriter>> reports;
       reports.emplace_back(report);
 
-      return is_valid_vcf_file(file, *validator, reports);
+      return vcf::is_valid_vcf_file(file, path.generic_string(), ebi::vcf::ValidationLevel::warning, 2, reports);
```

Comment:
Could you make the call to the `Ploidy` constructor explicit? It is not clear what the number means right now.

Applies to other classes, although in `ploidy_test.cpp` is it explicit.